resources/js/components/Board.js

```diff
@@ -310,14 +314,14 @@ export class Board extends React.Component {
               <input
                 name="text"
                 type="text"
-                placeholder="Text"
+                placeholder={this.state.heldTileText}
```

Comment:
You could probably make these `value` tags instead of `placeholder` so the user doesn't have to fully retype the word if they made a small typo near the end, for example.